backend/controllers/cellMove/cellMovePrisonerSearch.js

```diff
@@ -0,0 +1,65 @@
+const { alertFlagLabels, cellMoveAlertCodes } = require('../../shared/alertFlagValues')
+const { putLastNameFirst, formatLocation } = require('../../utils')
+
+module.exports = ({ prisonApi }) => async (req, res) => {
+  const {
+    user: { activeCaseLoad },
+  } = res.locals
+  const { keywords } = req.query
+
+  if (!keywords) {
+    const errors =
+      keywords === undefined
+        ? undefined
+        : [
+            {
+              href: '#keywords',
+              html: 'Enter a prisoner&#8217;s name or number',
+            },
+          ]
+    return res.render('cellMove/cellMovePrisonerSearch.njk', {
+      hasSearched: false,
+      errors,
+    })
+  }
+
+  const currentUserCaseLoad = activeCaseLoad && activeCaseLoad.caseLoadId
+
+  const context = {
+    ...res.locals,
+    requestHeaders: {
+      'Sort-Fields': 'lastName,firstName',
+      'Sort-Order': 'ASC',
+    },
+  }
+
+  const [prisoners] = await Promise.all([
+    prisonApi.getInmates(context, currentUserCaseLoad, {
+      keywords,
+      returnAlerts: 'true',
+    }),
+  ])
+
+  const results =
+    prisoners &&
+    prisoners.map(prisoner => ({
+      ...prisoner,
+      assignedLivingUnitDesc: formatLocation(prisoner.assignedLivingUnitDesc),
+      name: putLastNameFirst(prisoner.firstName, prisoner.lastName),
+      alerts: alertFlagLabels.filter(alertFlag =>
+        alertFlag.alertCodes.some(
+          alert =>
+            prisoner.alertsDetails && prisoner.alertsDetails.includes(alert) && cellMoveAlertCodes.includes(alert)
```

Comment:
You might be able to take advantage of optionals here. So instead of 
`prisoner.alertsDetails && prisoner.alertsDetails.includes(alert)`
Could you do
`prisoner.alertsDetails?.includes(alert)`

Reply:
Yes, copied code!